locales/cy/director-nationality.json

```diff
@@ -0,0 +1,10 @@
+{
+  "directoNationalityTitle": "Beth yw cenedligrwydd y cyfarwyddwr?",
```

Comment:
Typo in the field name, 'directo'
Are these the CH supplied welsh translations? If not I believe we are using the placeholder 'to be translated' in welsh files to avoid confusion on what has been translated.

Reply:
@jbestwickch Typo fixed
Yes they are welsh translations provided
https://companieshouse.atlassian.net/wiki/spaces/DACT/pages/4320821313/Update+translations

Reply:
My understanding from product was that we would not be adding welsh translations yet, future tickets being created for that.
https://companieshouse.atlassian.net/browse/DACT-1397

Reply:
@jbestwickch Welsh translations replaced with "to be translated"